greenwave/listeners/base.py

```diff
@@ -198,13 +204,15 @@ def _consume_message(self, message):
         self._publish_decision_change() and returns True or just
         returns False.
         """
-        raise NotImplementedError()
+        self.context: Context = TraceContextTextMapPropagator().extract(message)
 
     def _inc(self, messaging_counter):
         """Helper method to increase monitoring counter."""
         messaging_counter.labels(**self.monitor_labels).inc()
 
     def _publish_decision_update(self, decision):
+        self.app.logger.debug(f"Trace Context: {self.context}")
```

Comment:
Is this log needed at all?

```suggestion
```

Reply:
Seems like something left over from debugging while working on this change. @xpokor85 - Can you confirm?

Reply:
Removed